packages/plugins/max-aliases/test/index.spec.ts

```diff
@@ -103,6 +103,25 @@ describe('maxAliasesPlugin', () => {
     });
   });
 
+  it('counts __typename aliases against limit when ignoreFields is passed', async () => {
```

Comment:
Additionally, we can add a test to make sure the change is backward and forward compatible by checking if `__typename` is a default in the ignoreFields list.